src/python/install.sh

```diff
@@ -218,13 +306,71 @@ add_symlink() {
     fi
 }
 
+install_openssl11() {
+    local _prefix=$1
+    mkdir /tmp/openssl11
+    (
+    cd /tmp/openssl11
+    curl -L -f -O https://www.openssl.org/source/openssl-1.1.1w.tar.gz
```

Comment:
Hasn't this reached EOL? https://www.openssl.org/blog/blog/2023/09/11/eol-111/index.html#:~:text=available%20security%20fixes.-,OpenSSL%201.1.,of%20today%2C%2011th%20September%202023.


We shouldn't add a deprecated (/no longer supported) dependency 😓  Can we use `v3` ?

Reply:
Moving to v3 and running tests.

Reply:
Commit [baf1568](https://github.com/devcontainers/features/pull/830/commits/baf1568d80bb6f1eb73dd7e1b774b2fa5f3b662d) addresses this.